runtime/test/src/lib.rs

```diff
@@ -171,35 +170,11 @@ const STORAGE_ITEM_NAMES: &[&'static str] = &[
 pub struct SeedBridgeTransactionAndDropEthTransactions;
 
 impl frame_support::traits::OnRuntimeUpgrade for SeedBridgeTransactionAndDropEthTransactions {
```

Comment:
This looks incomplete.
Probably easier if we drop the `SeedBridgeTransactionAndDropEthTransactions` migration